src/main/java/org/apache/maven/plugins/assembly/internal/DebugConfigurationListener.java

```diff
@@ -0,0 +1,90 @@
+package org.apache.maven.plugins.assembly.internal;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.lang.reflect.Array;
+
+import org.codehaus.plexus.component.configurator.ConfigurationListener;
+import org.slf4j.Logger;
+
+/**
+ * Copy of deprecated {@link org.apache.maven.plugin.DebugConfigurationListener} updated to Slf4j.
+ */
+public class DebugConfigurationListener
+        implements ConfigurationListener
+{
+    private final Logger logger;
```

Comment:
Maybe logger can be initialized here instead of pass as argument.

Reply:
Yes, but I would rather throw this class out in next major. The benefit is very little.